components/event-chart/src/event-chart.ts

```diff
@@ -813,15 +813,20 @@ function formatRelativeTimestamp(timestamp: number): string {
   const min = sec * 60;
   const hour = min * 60;
   const day = hour * 24;
+  const decimals = 2;
   if (timestamp >= day) {
-    return `${timestamp / day}d`;
+    return `${roundUp(timestamp / day, decimals)} d`;
   } else if (timestamp >= hour) {
-    return `${timestamp / hour}h`;
+    return `${roundUp(timestamp / hour, decimals)} h`;
   } else if (timestamp >= min) {
-    return `${timestamp / min}min`;
+    return `${roundUp(timestamp / min, decimals)} min`;
   } else if (timestamp >= sec) {
-    return `${timestamp / sec}s`;
+    return `${roundUp(timestamp / sec, decimals)} s`;
   }
   return `${timestamp}ms`;
   // tslint:enable: no-magic-numbers
 }
+
+function roundUp(num: number, decimals: number): number {
+  return Math.round(10 ** decimals * num) / 10 ** decimals;
+}
```

Comment:
TIL:  ** is the exponentiation operator. 
Could you please add a jsdoc comment to the function to explain what it does :) 
It would also be amazing if you could add some unit tests for that function.

Reply:
ok, I'll add those